src/goto-programs/read_goto_binary.cpp

```diff
@@ -36,6 +36,18 @@ bool read_goto_binary(
     filename, dest.symbol_table, dest.goto_functions, message_handler);
 }
 
+optionalt<goto_modelt>
+read_goto_binary(const std::string &filename, message_handlert &message_handler)
+{
+  goto_modelt dest;
+
+  if(read_goto_binary(
+       filename, dest.symbol_table, dest.goto_functions, message_handler))
+    return {};
+  else
```

Comment:
Apply clang-format https://travis-ci.org/diffblue/cbmc/jobs/428936229#L510
Braces around `return {};` are recommended due to preceding multi-line condition.
`else` before `return dest;` could be omitted.